symengine/lib/symengine_wrapper.pyx

```diff
@@ -4399,16 +4365,20 @@ cdef class _Lambdify(object):
     cdef list out_shapes
     cdef readonly bint real
     cdef readonly int n_exprs
+    cdef readonly str order
```

Comment:
Make this a char?

Reply:
I tried that, but Cython seems to inject logic expecting an integer then.